app/models/expense.rb

```diff
@@ -135,7 +135,7 @@ def expense_type_unique_code=(code)
   end
 
   def vat_absent?
-    vat_amount.nil? || vat_amount == 0.0
+    vat_amount.nil? || vat_amount.zero?
```

Comment:
This is a large PR. IMO it would make sense to move the fixes for Lint/FloatComparison rubocop TODOs to a separate PR to keep it a little smaller as these are not related to specs.

Reply:
I'm happy to change this (and the other 3 cases) back. I realised after doing the `Lint/FloatComparison` bit that I should restrict it to just the spec files so these aren't any changes in the actual app in any of the other commits.

Reply:
👍